Assets/Scripts/PauseManager.cs

```diff
@@ -0,0 +1,58 @@
+﻿using UnityEngine;
+using UnityEngine.SceneManagement; // シーン遷移用
+
+public class PauseManager : MonoBehaviour
+{
+    public GameObject pauseCanvas; // PauseCanvasをアタッチ
+    public GameObject pauseButton; // PauseButtonをアタッチ
+
+    [Header("UI Groups")]
+    [SerializeField] private GameObject pauseMenuGroup;  // PauseMenuGroup (ボタンたち)
+    [SerializeField] private GameObject sensitivitySettingsGroup; // SensitivitySettingsGroup (感度設定)
```

Comment:
privateな変数名の頭にはアンダーバーをつける。（命名ルールの統一）

```suggestion
    [SerializeField] private GameObject _pauseMenuGroup;  // PauseMenuGroup (ボタンたち)
    [SerializeField] private GameObject _sensitivitySettingsGroup; // SensitivitySettingsGroup (感度設定)
```